worker/introspection/script.go

```diff
@@ -127,12 +129,11 @@ juju_stop_unit () {
     echo "usage: juju_stop_unit <unit-name> [<unit-name>...]"
     return 1
   fi
-  arr=("$@")
-  local -a args
-  for i in "${arr[@]}"; do
-    args+=("unit=$i")
+  args=""
+  for i in "$@"; do
+    args="$args unit=$i"
   done
-  juju_agent --post units action=stop "${args[@]}"
+  juju_agent --post units action=stop "$args"
```

Comment:
This doesn't actually have the same semantic.
However, maybe if we just avoid the "" around $args. It does look like:
```
$ tt () { args=""; for i in "$@"; do   args="$args unit=$i"; done; python3 -c 'import sys; print(sys.argv)' $args; }
$ tt a b
['-c', 'unit=a', 'unit=b']
```
Looks more like what we want to happen.